protocol/avdl/notify_service.avdl

```diff
@@ -0,0 +1,7 @@
+@namespace("keybase.1")
+protocol NotifyService {
+  import idl "common.avdl";
+
+  // sent when the service shuts down.
+  void shutdown() oneway;
```

Comment:
you might want to wait for an ACK otherwise you can shutdown before the service has a chance to send this out!